### ISSUE TITLE
[FR] Subscribe to issue

### ISSUE THREAD

**Issue by Offerel** [slapperwan/gh4a#1055]
It's there some way to subscribe and unsubscribe to an issue, without to comment in that issue?

**Comment by Zaseth** [comment]
I asked the same thing months ago but no reply. It seems like this repo died.

**Comment by maniac103** [comment]
The repo surely hasn't died (it's still there, isn't it?), it's just the maintainer doesn't have as much time as he used to, unfortunately.
This particular question seems to be a dupe of #215 to me, though.

**Comment by Offerel** [comment]
Hopefully we will find a way. Currently im using both, OctoDroid and the official mobile client. But the official client lacks the support for accessing the releases page, which i need really often. For this and some other functions, OctoDroid is a lot better.

**Comment by Zaseth** [comment]
It's annoying to open the tab in browser to click the sub/unsub button. @maniac103 I made many issues where things were missing or not working and they were neglected for 4+ months, hence why I'm saying it's dead.

**Comment by maniac103** [comment]
Yes, you made many issues, quite a few of which I fixed and thus are not present in the latest version anymore. I just left those untouched that need large refactorings (e.g. syntax highlighting issues), since I simply don't have enough time for that at this point.

**Comment by Zaseth** [comment]
I'm glad that you did, then I'll take my words back.

**Comment by Zaseth** [comment]
@maniac103 Oh and btw is it possible that you guys could publish a new version on the app store when the time is there?

**Comment by iexos** [comment]
Since its been almost 4 years since the last activity, I just want to say id love to (un)subscribe to issues from within the app :)

**Comment by Cwpute** [comment]
@Offerel if you could close this issue in favor of #215 which has been gathering information and useful discussions already ;)
